Misc/NEWS.d/next/Core and Builtins/2020-09-14-13-40-18.bpo-1635741.o6PrOV.rst

```diff
@@ -0,0 +1 @@
+Fix the documentation as per PEP 489.

```

Comment:
IMO the NEWS entry is not useful and should be removed.

Reply:
I totally agree - I actually thought it was a requirement for all changes.